generate_plugin_placeholder.rb

```diff
@@ -0,0 +1,58 @@
+require "clamp"
+require "erb"
+require "octokit"
+require_relative "git_helper"
+
+class GeneratePluginPlaceholder < Clamp::Command
```

Comment:
To avoid confusion with the plugin generator I suggest we scope the naming to include Docs

```suggestion
class GeneratePluginDocsPlaceholder < Clamp::Command
```

Reply:
WDTY for `GeneratePluginPlaceholderDoc`? `Docs` sounds the script generating non-single doc.